cor_graphical/verification/verification_report.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Verification Report: correlates_graphical'
+author: "Di Lu"
+date: "2021/03/27"
+output:
+  html_document: default
```

Comment:
Please remove the html_document output and keep only the pdf_document output. Remove html files that were generated.